_data/tasks.yml

```diff
@@ -23,4 +23,6 @@ toc:
   section:
   - title: Assigning Pods to Nodes
     path: /docs/tasks/administer-cluster/assign-pods-nodes/
+  - title: Using DNS Horizontal Autoscaling
```

Comment:
I think Tutorial, not Task, is probably the best category for this topic. Even so, it might be best to leave it under Tasks/Administering a Cluster for now. Here's a portion of the plan for topics that will eventually be under Tutorials:

Keeping Your Production Application Healthy
    Running a Canary Deployment Using Labels
    Separating Tenant Workloads Using Namespaces
    Syncing a Github Repo by Adding a support Container
    Restarting Unhealthy Containers Using Readiness and Liveness Probes
    Reserving Capacity for a Pod Using Resource Reservations
    Gracefully Shutting Down a Container by Listening to the Termination Notice
    Updating Container Exit Status Using a Termination Message

Recommended Release Practices
    Expanding Environment Variables in Configuration
    Scripting with kubectl

If you think this topic fits well under "Keeping Your Production Application Healthy" or under "Recommended Release Practices," then I would suggest putting it there as a tutorial. Or if you envision another tutorial category, you could create the category and put this topic under it. But if you prefer to leave it under tasks for now, I think that's fine.